scripts/variantstore/scripts/run_avro_query_for_sample_info.py

```diff
@@ -32,12 +32,27 @@ def get_number_of_partitions(dataset_name, project_id):
     return math.ceil(max_table_num[0])
 
 
-def construct_sample_info_avro_queries(call_set_identifier, dataset_name, project_id, avro_prefix):
+def construct_sample_info_avro_queries(call_set_identifier, dataset_name, project_id, avro_prefix, new_sample_cutoff=None):
```

Comment:
Not sure why this sample_info method is thinking in terms of superpartitions... this seems to be greatly complicating things.

Reply:
oh I may have forgotten, does this does need to be in superpartitions for mapping reasons?

Reply:
I am worried that my explanation that I thought was very clear on line 38 is not clear or that we are talking about different things.